### PR TITLE
New elections galore

### PR DESCRIPTION
**Pull request by nbdavies**
As a result of finding elections on the WEC site that we didn't have in the OpenElections database...
* Fetched new files.
* Modified parser to handle files with single-line headers.
* Parsed newly discovered elections.

### CODE REVIEW

File: parser.py [hunk 1]
```diff
@@ -52,10 +54,20 @@ def process_xls_2002_to_2010(sheet):
...
+        elif colA in (non_data_row_first_column_values) or 'rows selected.' in str(colA):
             continue
         
         # not header nor blank: assume this is a data row
+        if parties is None:  # If we didn't find the parties on their own row, they were tacked onto the candidate names
+            clean_candidates = []
+            parties = []
+            for candidate in candidates:
+                if ' ' in candidate:
+                    candidate, party = candidate.strip().rsplit(' ', 1)
```
Comment (davipo) [inline parser.py:65]: This won't work for parties that contain a space ("Wisconsin Greens").

File: parser.py [hunk 2]
```diff
@@ -37,6 +38,7 @@ def collect_columns(row, start_col):
 def process_xls_2002_to_2010(sheet):
     """Return list of records from spreadsheet in 2002-2010 format"""
     results = []
+    parties = None
```
Comment (davipo) [inline parser.py:40]: Move to line after candidates are collected, before 'continue' statement, to reset parties for a new block (with new header lines). Some files have multiple blocks (maybe ones with single-line headers don't?).

('parties' can be left undefined if no first header is found -- we will see an error for undefined candidates first.)

**Review by davipo — COMMENTED** [review]
I've rewritten this to handle issues I commented (and simplify). Will test soon.

**Comment by nbdavies** [comment]
This PR has been replaced with #15, which has all the relevant fixes and new input/output files and none of the hacky stuff and merge conflicts.

**Pull request closed without merging**